src/roiextractors/testing.py

```diff
@@ -11,6 +11,29 @@
 inttype = (int, np.integer)
 
 
+import numpy as np
+
+from roiextractors import NumpyImagingExtractor
+
+
+def generate_dummy_imaging_extractor(
+    num_frames: int = 100,
+    rows: int = 10,
+    columns: int = 10,
+    num_channels: int = 1,
+    sampling_frequency: float = 30000,
+    dtype="int16",
```

Comment:
Our data is usually unsigned as well - though, yes, it is often `uint16` instead of `uint8` (mostly because it is usually grayscale).

Reply:
Also can you add annotation typing? Either strict string input, or union of string and numpy dtype?

Reply:
Done.